test/src/main/java/org/apache/accumulo/test/metrics/fate/ZooLockTest.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.accumulo.test.fate.zookeeper;
+package org.apache.accumulo.test.metrics.fate;
```

Comment:
This should not be relocated to a metrics subpackage. It is not related to metrics.